src/site/content/en/blog/periodic-background-sync/index.md

```diff
@@ -69,7 +73,7 @@ Status
 2. Create initial draft of specification
 </td>
 <td markdown="block">
-In Progress
+[In Progress](https://github.com/WICG/BackgroundSync/blob/master/explainers/periodicsync-explainer.md)
```

Comment:
```suggestion
<a href="https://github.com/WICG/BackgroundSync/blob/master/explainers/periodicsync-explainer.md" rel="noopener">In Progress</a>
```
MD won't parse without extra whitespace, which can be fussy, so switched to HTML.